packages/flutter_android_lifecycle/android/src/main/java/io/flutter/embedding/engine/plugins/lifecycle/FlutterLifecycleAdapter.java

```diff
@@ -0,0 +1,22 @@
+package io.flutter.embedding.engine.plugins.lifecycle;
+
+import android.arch.lifecycle.Lifecycle;
```

Comment:
@blasten this was moved to the androidx namespace, do we expect this to always work well for projects that migrated to AndroidX? (whether plugins are built as AARs or not)

Reply:
If possible, I think we should use androidx here. The current logic to build the plugins as AARs makes the build pretty slow.

Reply:
Changed to androidx references and enabled jetifier for the example project.